README

```diff
@@ -0,0 +1,32 @@
+python-tx-tftp
```

Comment:
I think it does need to be checked in. Running `python setup.py ...` will look for `README` and, if it's not found, will not build it. A step in `setup.py` *could* generate it, but there is no guarantee that the environment in which it runs will have `pandoc` installed. It's a non-Python dependency so we can't even ask `setuptools` to get it for us.

I've read that PyPI supports Markdown, but I couldn't get it to work in practice. Once that works, or when someone figures out the way to make it work, we can get rid of `README`.